mujinvisioncontrollerclient/visioncontrollerclient.py

```diff
@@ -188,6 +188,15 @@ def IsWaitingResponse(self):
         assert self._commandsocket is not None
         return self._commandsocket.IsWaitingReply()
 
+    def WaitForGetLatestDetectionResultImages(self, timeout=2.0):
+        # type: (float) -> Dict
+        """Waits for response to GetLatestDetectionResultImages command
+
+        Args:
+            timeout (float, optional): Time in seconds after which the command is assumed to have failed. (Default: 2.0)
+        """
+        return self._WaitForResponse(recvjson=False, timeout=timeout)
+
```

Comment:
Why are we moving the function?

Reply:
I agree that it fits better at the old place. But because we want to generate those clients - its inconvenient to put hard coded functions as `WaitForGetLatestDetectionResultImages` in between functions that are auto generated.

Reply:
Sure, I understand